package.json

```diff
@@ -33,6 +33,7 @@
     "google-protobuf": "^3.6.1"
   },
   "devDependencies": {
+    "@grpc/grpc-js": "^1.2.0",
```

Comment:
This is probably best expressed in the README rather than this package specifying it as a dev dependency as that might not get noticed.

Reply:
I think adding it as a dev dependency was intended to make it easier to test the generated types, but it's not required by the package so I have remove it. 
I've also added a compatibility note to the README.